### PR TITLE
feat(apm): include get and list commands for APM applications

### PR DESCRIPTION
**Pull request by zlesnr**
resolves: #79

### CODE REVIEW

File: internal/apm/command.go [hunk 1]
```diff
@@ -106,6 +106,56 @@ deployment.
...
+	Use:   "list",
+	Short: "List the New Relic appliations",
+	Long: `List the New Relic application
+
+The list command performs a query for all APM applications.
+`,
+	Example: "newrelic apm list",
+	Run: func(cmd *cobra.Command, args []string) {
+		client.WithClient(func(nrClient *newrelic.NewRelic) {
+			d, err := nrClient.APM.ListApplications(&apm.ListApplicationsParams{})
```
[the remark this answers is not in the document]
Comment (zlesnr) [reply internal/apm/command.go]: Should this hang off the entities command then?  If we're just calling out to the entities methods, it seems that we already have a command for this purpose.

File: internal/apm/command.go [hunk 2]
```diff
@@ -106,6 +106,56 @@ deployment.
...
 }
 
+var apmGetApplication = &cobra.Command{
+	Use:   "get",
+	Short: "Get a New Relic application by ID",
+	Long: `Get a New Relic application by ID
+
+The get command performs a query for an APM application by ID.
+`,
+	Example: "newrelic apm get --applicationID <appID>",
```
[the remark this answers is not in the document]
Comment (zlesnr) [reply internal/apm/command.go]: We talked about this one.

File: internal/apm/command.go [hunk 3]
```diff
@@ -106,6 +106,56 @@ deployment.
...
+}
+
+var apmListApplications = &cobra.Command{
+	Use:   "list",
+	Short: "List the New Relic appliations",
+	Long: `List the New Relic application
+
+The list command performs a query for all APM applications.
+`,
+	Example: "newrelic apm list",
```
[the remark this answers is not in the document]
Comment (zlesnr) [reply internal/apm/command.go]: I'm going to defer implementing list for now.

File: internal/apm/command.go [hunk 4]
```diff
@@ -7,14 +7,18 @@ import (
...
+	"github.com/newrelic/newrelic-client-go/pkg/entities"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
 var (
-	apmApplicationID   int
-	deploymentRevision string
-	deploymentID       int
+	apmApplicationAccount string
```
Comment (jthurman42) [inline internal/apm/command.go]: This should be `apmApplicationAccountID`

**Pull request merged**